lib/qless/middleware/retry_exceptions.rb

```diff
@@ -48,19 +48,18 @@ def on_retry_callback
         @on_retry_callback ||= DEFAULT_ON_RETRY_CALLBACK
       end
 
-      def exponential(base, options = {})
+      # If`factor` is omitted it is set to `delay_s` to reproduce legacy
+      # behavior.
+      def exponential(delay_s, options={})
```

Comment:
> How do we want to balance clarity vs. verbosity?

Yep, always a question of tradeoffs.  In this particular case, `delay_seconds` does not seem particularly verbose, IMO.  In addition, while I see the appeal of a convention of using a `_s` suffix to indicate seconds I have never seen that convention in any code at Moz or in any open source Ruby code.  I think it's worth having a discussion as a team about at some point (and if there is agreement, we could apply it to the code bases we own like vanguard and packrat) but given qless's nature as something that is used by other teams and outside the company as an open source project, I think we should error on the side of the more explicit `delay_seconds`.